automobile/car-library.md

```diff
@@ -130,13 +454,81 @@ All these functions provide important physical characteristics from the car.
 #### `wbu_car_get_wheel_encoder`
 #### `wbu_car_get_wheel_speed`
 
+%tab-component
+
+%tab "C"
+
 ```c
 #include <webots/car.h>
 
+typedef enum {
+  WBU_CAR_WHEEL_FRONT_RIGHT,
+  WBU_CAR_WHEEL_FRONT_LEFT,
+  WBU_CAR_WHEEL_REAR_RIGHT,
+  WBU_CAR_WHEEL_REAR_LEFT,
+  WBU_CAR_WHEEL_NB
+} wbu_car_wheel_index;
+
 double wbu_car_get_wheel_encoder(int wheel_index);
 double wbu_car_get_wheel_speed(int wheel_index);
 ```
 
+%tab-end
+
+%tab "C++"
+
+```cpp
+#include <webots/Car.hpp>
+
+namespace webots {
+  class Car : public Driver {
+    double getWheelEncoder(int wheel_index);
```

Comment:
missin declaration of enum for WHEEL_FRONT_RIGHT, etc.

same for Python and Java.